webpack.config.js

```diff
@@ -23,13 +19,10 @@ var config = {
       },
       {
         test: /\.css$/,
-        loader: ExtractTextPlugin.extract("style-loader", "css-loader"),
+        loader: "style-loader!css-loader",
       },
     ],
   },
-  plugins: [
```

Comment:
with removing this, did you also remove the css file definition in the index.html file?